client/src/components/publications/publication/Publication.js

```diff
@@ -160,9 +167,9 @@ const Publication = ({ pub }) => {
             {
               isHovering
             && (
-            <StyledButtonGroup className="float-right">
+            <StyledButtonGroup onClick={childCallback} className="float-right">
```

Comment:
curious: do we actually need an `onClick` here? If the whole purpose of this is to prevent the click event from   being propagated up the DOM, why not just not have the `onClick` event?

Reply:
Cause I have the onClick event on its parent div (line 147), and I don't want to trigger that event when its children div are clicked (line 171 and 172) cause the children div has their own onClick event. 
Not sure if it is the correct way to do it but I found this solution from here. 
https://stackoverflow.com/questions/37568550/react-prevent-event-trigger-on-parent-from-child